AdyenActions/Components/SDK/TwintSDKActionComponent.swift

```diff
@@ -235,6 +245,17 @@ import Foundation
         private func cleanup() {
             pollingComponent?.didCancel()
         }
+        
+        private func sendThirdPartyErrorEvent(with message: String?, componentName: String) {
```

Comment:
Do we need to add extra method to send events for each domain?
I see we have an enum that has clear domain split `AnalyticsConstants.ErrorCode.thirdPartyError` so could we use a generic `sendError(...)` or `sendEvent(...)` method if we have so?